pkg/controller/datavolume-controller.go

```diff
@@ -710,6 +710,11 @@ func (c *DataVolumeController) updateDataVolumeStatus(dataVolume *cdiv1.DataVolu
 		switch pvc.Status.Phase {
 		case corev1.ClaimPending:
 			dataVolumeCopy.Status.Phase = cdiv1.Pending
```

Comment:
I think it would be helpful to comment here why a pending PVC could get the succeeded annotation.  It's not obvious what special case we're dealing with here.

Reply:
Done